core/src/test/java/nl/nn/adapterframework/jdbc/dbms/DbmsSupportTest.java

```diff
@@ -612,10 +616,67 @@ public void testQueueHandling() throws Exception {
 									assertEquals(41,rs2.getInt(1));	// find the second record
 								}
 							}
+							workConn2.rollback();
 						}
+					} else {
```

Comment:
Het is wel een enorme methode geworden, is het mogelijk de methode op te splitsen in hasSkipLockedFunctionality en doesNotHaveSkipLockedFunctionality